nrf802154_sniffer/custom_dissector.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/nrf802154_sniffer/nrf802154_sniffer.py b/nrf802154_sniffer/nrf802154_sniffer.py
+index e8dba56..cc42ccb 100755
+--- a/nrf802154_sniffer/nrf802154_sniffer.py
++++ b/nrf802154_sniffer/nrf802154_sniffer.py
+@@ -61,8 +61,8 @@ from serial.tools.list_ports import comports
+ class Nrf802154Sniffer(object):
+ 
+     # Various options for pcap files: http://www.tcpdump.org/linktypes.html
+-    #DLT='user'
+-    DLT='802.15.4'
++    DLT='user'
```

Comment:
The correct way to solve this is to request a dedicated DLT for this header, like we did with the BLE sniffer. Have a look at https://www.tcpdump.org/linktypes.html for details, and https://www.tcpdump.org/linktypes/LINKTYPE_NORDIC_BLE.html for an example.

Reply:
I think that it would be a great improvement but before requesting a dedicated DTL we should make this sniffer more mature.